dev-guide/build.md

```diff
@@ -0,0 +1,29 @@
+# Build
+
+## Overview
+
+If you want to dive into TiDB project, change some codes and send us PR, you may follow this guide to build TiDB project.
+
+You can see [user build guide](../op-guide/build.md) to check platforms and prerequisites first. 
```

Comment:
+See [user build guide](../op-guide/build.md) to check the supported platforms and prerequisites first.